projects/espace/api/migrations/Version20251231142632.php

```
@@ -0,0 +1,32 @@
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20251231142632 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)